packages/nvidia_nat_langchain/tests/test_langgraph_workflow.py

```
@@ -0,0 +1,203 @@
+
+    def test_multi_key_dict_raises(self):
+        """Test that multiple node keys raises an error."""
+        raw = {
+            "node_a": {
+                "messages": [AIMessage(content="a")]
+            },
+            "node_b": {
+                "messages": [AIMessage(content="b")]
+            },
+        }
+        with pytest.raises(Exception):
```

Ruff (0.15.1)

[warning] Do not assert blind exception: `Exception`

(B017)